src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/custom.py

```diff
@@ -36,51 +44,116 @@ def cli_topic_list(
     return client.list_by_subscription()
 
 
-def cli_eventgrid_event_subscription_create(
+def cli_topic_create_or_update(
+        client,
+        resource_group_name,
+        topic_name,
+        location,
+        tags=None):
+    async_topic_create = client.create_or_update(
+        resource_group_name,
+        topic_name,
+        location,
+        tags)
+
+    created_topic = async_topic_create.result()
+
+    return created_topic
```

Comment:
Why was this custom command needed? Simply calling the sdk command looks like it will do the same.